src/tests/backend/specs/api/httpRouteSessionCreation.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+const assert = require('assert').strict;
+const common = require('../../common');
+const settings = require('../../../../node/utils/Settings');
+const fs = require('fs');
+let agent;
+
+const shouldCreateSession = [
+  '/p/foo',
+  '/p/foo/export/html',
+];
+const shouldNotCreateSession = [
+  '/',
+  '/api/',
+  '/favicon.ico',
+  '/locales.json',
+  '/pluginfw/plugin-definitions.json',
+  '/static/js/pad.js',
+  '/stats/',
+];
+
+const getDatabaseSize = () => {
```

Comment:
This is fragile. Instead, implement the [`Store.length() method`](https://www.npmjs.com/package/express-session#storeallcallback) in the SessionStore class in `SessionStore.js`.

Reply:
I made an attempt at this but it didn't work.  I'm unsure how to access ``Store`` as it's the parent of ``SessionStore`` (which is extending ``Store``)

Reply:
If I read it correctly, we would need to add our own `length` method and increase/decrease it on set/destroy